documents/resources.py

```diff
@@ -0,0 +1,24 @@
+import documents.state_machine as state_machine
+import utils
+from flask import jsonify, request, Blueprint
+
+documents = Blueprint("documents", __name__, template_folder="templates")
+
+@documents.route('', methods=['POST'])
+@utils.server_error_check
+@utils.authenticate_user
+def upsert_documents(payload):
+    user_id = payload['userid']
```

Comment:
why no underscore here?

Reply:
didn't add an underscore in the payload part at the start, that's why it's "userid"

Reply:
then add there too